Formula/gopass.rb

```diff
@@ -0,0 +1,58 @@
+class Gopass < Formula
+  desc "The slightly more awesome Standard Unix Password Manager for Teams."
+  homepage "https://www.justwatch.com/gopass"
+  url "https://www.justwatch.com/gopass/releases/1.0.2/gopass-1.0.2.tar.gz"
+  sha256 "a2c64329355f72527a1e269b3ad97cd2d37012ebf283210ef947c10657d8f650"
+  head "https://github.com/justwatchcom/gopass.git"
+
+  depends_on "go" => :build
+
+  depends_on :gpg => :run
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/justwatchcom/gopass").install buildpath.children
+
+    cd "src/github.com/justwatchcom/gopass" do
+      ENV["PREFIX"] = prefix
+      system "make", "install"
+    end
+
+    system bin/"gopass completion bash > bash_completion.bash"
+    system bin/"gopass completion zsh > zsh_completion.zsh"
+    bash_completion.install "bash_completion.bash"
+    zsh_completion.install "zsh_completion.zsh"
+  end
+
+  def caveats; <<-EOS.undent
```

Comment:
`caveats` should be for Homebrew-specific installation information. General usage information is not suitable there.